experiment/measurer/measure_manager.py

```diff
@@ -755,51 +700,294 @@ def get_pool_args(measurers_cpus, runners_cpus):
+        self.response_queue_topic_id = f'response-queue-topic-{self.experiment}'
+        self.response_queue_subscription_id = f"""response-queue-subscription-
+            {self.experiment}"""
+        self.subscriber_client = pubsub_v1.SubscriberClient()
+        self.publisher_client = pubsub_v1.PublisherClient()
+        self.subscription_path = self.subscriber_client.subscription_path(
+            self.project_id, self.response_queue_subscription_id)
+        self.measurers_cpus = measurers_cpus
+
+    def initialize_queues(self) -> Tuple[str, str]:
+        request_queue_topic_path = self.publisher_client.topic_path(
+            self.project_id, self.request_queue_topic_id)
+        request_queue_topic = self.publisher_client.create_topic(
+            request={'name': request_queue_topic_path})
+        response_queue_topic_path = self.subscriber_client.topic_path(
+            self.project_id, self.response_queue_topic_id)
+        response_queue_topic = self.publisher_client.create_topic(
+            request={'name': response_queue_topic_path})
+        return request_queue_topic.name, response_queue_topic.name
+
+    def _create_response_queue_subscription(self):
+        """Creates a new Pub/Sub subscription for the response queue."""
+        topic_path = self.response_queue_topic_id
+        subscription = self.subscriber_client.create_subscription(request={
+            'name': self.subscription_path,
+            'topic': topic_path
+        })
+        logger.info(f'Subscription {subscription.name} created successfully.')
+
+        return self.subscription_path
+
+    def start_workers(self, request_queue, response_queue):
+        self._create_response_queue_subscription()
+        if not self.measurers_cpus:
+            self.measurers_cpus = multiprocessing.cpu_count()
+            logger.info(
+                'Number of measurer CPUs not passed as argument. using %d',
+                self.measurers_cpus)
+        with multiprocessing.Pool(processes=self.measurers_cpus) as pool:
+            config = {
+                'request_queue_topic_id': self.request_queue_topic_id,
+                'response_queue_topic_id': self.response_queue_topic_id,
+                'region_coverage': self.region_coverage,
+                'project_id': self.project_id,
+                'experiment': self.experiment,
+            }
+            google_cloud_worker = measure_worker.GoogleCloudMeasureWorker(
+                config)
+
+            # Since each worker is going to be in an infinite loop, we dont need
+            # result return. Workers' life scope will end automatically when
+            # there are no more snapshots left to measure.
+            logger.info('Starting measure worker loop for %d workers',
+                        self.measurers_cpus)
+            for _ in range(self.measurers_cpus):
+                _result = pool.apply_async(
+                    google_cloud_worker.measure_worker_loop)
```

Comment:
Why we do need to store the return value to `_result`?

Reply:
We actually don't need it, I will remove that! thx